### DISCUSSION TITLE
armv7 vs linux-armv7

### DISCUSSION THREAD

**Discussion by stianholsen** [Vector35/binaryninja-api#3039] [Architectures] [answered]
What's the difference between armv7 and linux-armv7? I see that some ELF files return armv7 and some return linux-armv7 when running bv.platform

**Comment by psifertex** [comment, marked as answer]
`Platform` normally contains a platform such as `freebsd`,`linux`, `mac`, or `windows`. You can also define your own. These platforms include some things like standard syscalls for example. If you get just a platform of `armv7` it means the binary is being treated as a bare armv7 architecture without knowing the OS associated with it. 

If you actually look in your install folder (or `Binary Ninja.app/Contents/Resources` on MacOS), you'll see a `types` folder (which is a bit mis-named) as it contains the platform type information. Please don't make changes to those files though as they are [overwritten on update](https://docs.binary.ninja/getting-started.html#binary-path) (see the warning).
